driver/src/test/scala/org/mongodb/scala/AggregateObservableSpec.scala

```diff
@@ -18,36 +18,27 @@ package org.mongodb.scala
 
 import java.util.concurrent.TimeUnit
 
-import scala.concurrent.duration.Duration
-
-import com.mongodb.async.client.{AggregateIterable, MongoIterable}
-
+import com.mongodb.reactivestreams.client.AggregatePublisher
 import org.mongodb.scala.model.Collation
 import org.scalamock.scalatest.proxy.MockFactory
 import org.scalatest.{FlatSpec, Matchers}
 
-class AggregateObservableSpec extends FlatSpec with Matchers with MockFactory {
+import scala.concurrent.duration.Duration
 
-  def observer[T]: Observer[T] = new Observer[T]() {
```

Comment:
For the tests we no longer need an Observable we can just test the publishers method is called.

Reply:
ack